frontend/src/components/ui/quick-filter-dropdown.tsx

```diff
@@ -50,50 +76,73 @@ export default function QuickFilterDropdown({
       <DropdownMenuContent className="rounded-[8px] p-0">
         <DropdownMenuLabel className="px-4 py-[12px]">{text}</DropdownMenuLabel>
         <DropdownMenuSeparator />
-        <div className="p-4">
-          <RadioGroup
-            defaultValue={selected}
-            className="flex flex-col gap-[12px]"
-            onValueChange={(v) => setSelected(v)}
+        {isCheckbox ? (
+          <div
+            onClick={() => onChange([{ value: [], key: filterFieldName }])}
+            className="ml-auto flex cursor-pointer justify-end px-4 pt-2 text-sm font-semibold text-[#006064]"
           >
-            <div className="flex items-center space-x-2">
-              <RadioGroupItem
-                value="*"
-                id="option-one"
-                className="border-gray-300 text-accent data-[state=checked]:border-accent"
-                onSelect={(e) => {
-                  console.log(e);
-                }}
-              />
-              <Label
-                htmlFor="option-one"
-                className="text-sm font-medium leading-none text-gray-500"
-              >
-                All
-              </Label>
-            </div>
-            {items.map((item, i) => (
-              <div
-                className="flex items-center space-x-2"
-                key={`${item.text}-${i}`}
-              >
-                <RadioGroupItem
-                  className="border-gray-300 text-accent data-[state=checked]:border-accent"
-                  value={item.value}
-                  id={`${item.text}-${i}`}
-                />
-                <Label
-                  htmlFor={`${item.text}-${i}`}
-                  className="text-sm font-medium leading-none text-gray-500"
+            Clear filter
```

Comment:
I think this clear filter is filling the entire width, which ends up meaning that if i click outside of it, it clears all my fiilters which is kind of anoying
![image](https://github.com/user-attachments/assets/f075e35c-2df5-479b-9bed-1a88c5b93fc6)